site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -756,6 +756,14 @@ function () use ($csv_file) {
         $this->core->redirect($return_url);
     }
 
+    /**
+     * Returns Custom marks access boolean
+     * @Route("/courses/{_semester}/{_course}/gradeable/{gradeable_id}/getCustomMarksAccess", methods={"GET"})
+     */
+    public function getCustomMarksAccess($gradeable_id) {
```

Comment:
Currently silent_edit permissions is defined as instructor only in Access.php. This makes sense, because only instructors should be able to edit a rubric. However, full and limited access graders should be able to leave custom marks. I believe this should be changed to say if the current user's group < GROUP::STUDENT, return true, else return false

Reply:
@jchulton Made the changes for it.